src/all.js

```diff
@@ -4,13 +4,19 @@ import TimeoutDialog from './components/timeout-dialog/timeout-dialog';
 import UserResearchBanner from './components/user-research-banner/user-research-banner';
 import { nodeListForEach } from './common';
 import SessionActivityService from './components/timeout-dialog/session-activity-service';
+import HmrcPrintLink from './components/hmrc-print-link/hmrc-print-link';
 
 function initAll() {
   const $AccountMenuSelector = '[data-module="hmrc-account-menu"]';
   if (document.querySelector($AccountMenuSelector)) {
     new AccountMenu($AccountMenuSelector).init();
   }
 
+  const $HmrcPrintLink = document.querySelector('a[data-module="hmrc-print-link"]');
+  if ($HmrcPrintLink) {
+    new HmrcPrintLink($HmrcPrintLink, window).init();
+  }
```

Comment:
```suggestion
  const $HmrcPrintLinks = document.querySelectorAll('a[data-module="hmrc-print-link"]');
  nodeListForEach($HmrcPrintLinks, ($HmrcPrintLink) => {
    new HmrcPrintLink($HmrcPrintLink, window).init();
  });
```

just in case for some reason someone puts two on one page, like at top and bottom of a long page maybe